fiducialMarker/fiducialMarkerTest.py

```diff
@@ -0,0 +1,40 @@
+from cv2 import aruco
+from fiducialMarker import FiducialMarker
+import numpy
+import unittest
+
+
+class TestFiducialMarker(unittest.TestCase):
+
+    @staticmethod
+    def test_positive_generate_marker():
+        # img_true attained from output in python3 shell of following inputs:
+        # dictionary = aruco.getPredefinedDictionary(aruco.DICT_4X4_50)
+        # img_true = aruco.drawMarker(dictionary, 0, 6)
+        img_true = numpy.array([[0,   0,   0,   0,   0,   0],

```

Comment:
So this marker matrix is static for the current predefined dictionary?
If we are sticking to this default array dictionary, then these tests definitely work. I think once we get the markers settled, we can make tests for those too :)
Also, love your formatting.